screen2.0/src/app/search/_ccredetails/transcriptionatccre.tsx

```diff
@@ -0,0 +1,222 @@
+      peakId: string
+      genes: [{
+        geneName: string
+        locusType: string
+      }]
+  }[]
+}
+
+export type PeakVars = {
+  coordinates: {
+    chromosome: string
+    start: number
+    stop: number
+  }
+}
+
+export const OverlappingPeaks: React.FC<PeakVars> = ({ coordinates }) => {
+  const [selectedRow, setSelectedRow] = useState<PeakRow | null>(null);
+  const [sort, setSort] = useState<"byValue" | "byTissueMax" | "byTissue">("byValue")
+  const handleRowClick = (row: PeakRow) => {
+    setSelectedRow(row);
+  };
+
+  const { loading, error, data } = useQuery<PeakData, PeakVars>(TSS_RAMPAGE_PEAKS,
+    {
+    variables: {
+      coordinates: coordinates,
+    },
+    fetchPolicy: "cache-and-network",
+    nextFetchPolicy: "cache-first",
+    client,
+  })
+
+  const peakData = useMemo(() => {
+    return data?.gettssRampagePeaks.map((x) => ({
+      peakType: x.peakType,
+      start: x.start,
+      chrom: x.chrom,
+      end: x.end,
+      peakId: x.peakId,
+      gene: x.genes[0]?.geneName || "N/A",
+      locustype: x.genes[0]?.locusType || "N/A",
+    })) || [];
+  }, [data]);
+
+  useEffect(() => {
+    if (peakData.length > 0) {
+      setSelectedRow(peakData[peakData.length-1]);
+    }
+  }, [peakData]);
+
+  type PeakRow = {
+    peakType: string;
+    start: number;
+    chrom: string;
+    end: number;
+    peakId: string;
+    gene: string;
+    locustype: string;
+  };
+
+  type RampageData = {
+    tssrampageQuery: {
+      start: number,
+      organ: string,
+      strand: string,
+      peakId: string,
+      biosampleName: string,
+      biosampleType: string,
+      bisampleSummary: string,
+      peakType: string,
+      expAccession: string,
+      value: number,
+      end: number,
+      chrom: string,
+      genes: [{
+        geneName: string
+        locusType: string
+      }]
+  }[]
+  }
+
+  type RampageVars = {
+    peak: string;
+  }
+
+  const { loading: rampageloading, error: rampageerror, data: rampagedata } = useQuery<RampageData, RampageVars>(
+    TSS_RAMPAGE_QUERY, 
+    {
+      variables: {
+        peak: selectedRow?.peakId || "",
+      },
+      skip: !selectedRow,
+      fetchPolicy: "cache-and-network",
+      nextFetchPolicy: "cache-first",
+      client,
+    }
+  )
+
+  const rampageData = useMemo(() => {
+    if (!rampagedata) return [];
+  
+    return rampagedata.tssrampageQuery.map((t) => ({
+      value: t.value,
+      peakId: t.peakId,
+      biosampleType: t.biosampleType,
+      name: t.biosampleName,
+      locusType: t.genes[0].locusType,
+      expAccession: t.expAccession,
+      start: String(t.start),
+      end: String(t.end),
+      chrom: t.chrom,
+      peakType: t.peakType,
+      organ: t.organ,
+      strand: t.strand,
+      tissue: t.organ,
+    }));
+  }, [rampagedata]);
+  
+  return loading ? (
+    <LoadingMessage />
+  ) : error ? (
+    <ErrorMessage error={error} />
+  ) : (
+    <Grid2 container spacing={3} sx={{ mt: "0rem", mb: "2rem" }}>
```

Comment:
This grid and all child grid elements can be removed and replaced by one parent stack element. I was previously very overeager to use grid which is why its in a bunch of these files. Stack is best for one dimensional layouts like this one https://mui.com/material-ui/react-stack/